CMakeLists.txt

```
@@ -495,6 +519,7 @@ message(STATUS "    Code for char * ...... ${URIPARSER_BUILD_CHAR}")
 message(STATUS "    Code for wchar_t * ... ${URIPARSER_BUILD_WCHAR_T}")
 message(STATUS "    Tools ................ ${URIPARSER_BUILD_TOOLS}")
 message(STATUS "    Test suite ........... ${URIPARSER_BUILD_TESTS}")
+message(STATUS "    Fuzzing tests ........ ${URIPARSER_BUILD_FUZZERS}")
```

```suggestion
message(STATUS "    Fuzzers .............. ${URIPARSER_BUILD_FUZZERS}")
```

(Because these are not tests to me.)

---

Fixed